### DISCUSSION TITLE
Annoying page scroll popup at prompthero.com

### DISCUSSION THREAD

**Discussion by deadman3000** [uBlockOrigin/uAssets#24133] [General]
I don't mind the occasional popup but this one is egregious for continuously popping up each time you scroll down a few lines on their images page in an attempt to get you to to sign up (no doubt so they can harvest all your lovely information).
The site is at 
https://prompthero.com/stable-diffusion-prompts

The Javascript appears to be injected from 
prompthero.s3.amazonaws.com/backgrounds/abstract-spaceship-prompthero-1.webp

Blocking this does not work. The only way I have got it to stop is by using the zapper tool but once the page is reloaded it comes right back. The normal option of selecting the element to block does not work.

Can it be blocked?

**Comment by gorhill** [comment]
```adblock
prompthero.com##.sign-up-modal
prompthero.com##.modal-backdrop
prompthero.com##body:style(overflow:initial!important)
```

You can just submit an issue through the _chat_ icon next time.
